src/apiv2.js

```diff
@@ -3,9 +3,10 @@ const lib = require("./lib");
 const cors = require("cors");
 const sanitize = require("mongo-sanitize");
 const constants = require("./constants");
+const credentials = require("./../credentials.json");
 
 const Redis = require("ioredis");
-const redisClient = new Redis();
+const redisClient = new Redis(credentials.redisUrl ?? credentials.redisUrl);
```

Comment:
`credentials.redisUrl ?? credentials.redisUrl` is redundant because `credentials.redisUrl` and `credentials.redisUrl` are the same thing.

see this MDN page if you're not familiar with how `??` works https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
thanks for the quick reply and sorry for that stupid mistake